extractor/src/main/java/org/schabi/newpipe/extractor/services/media_ccc/extractors/MediaCCCLiveStreamExtractor.java

```diff
@@ -134,49 +141,59 @@ public boolean isUploaderVerified() throws ParsingException {
     @Nonnull
     @Override
     public String getUploaderAvatarUrl() {
-        return "";
+        return EMPTY_STRING;
     }
 
     @Nonnull
     @Override
     public String getSubChannelUrl() {
-        return "";
+        return EMPTY_STRING;
     }
 
     @Nonnull
     @Override
     public String getSubChannelName() {
-        return "";
+        return EMPTY_STRING;
     }
 
     @Nonnull
     @Override
     public String getSubChannelAvatarUrl() {
-        return "";
+        return EMPTY_STRING;
     }
 
     @Nonnull
     @Override
     public String getDashMpdUrl() throws ParsingException {
-        return "";
+        for (int s = 0; s < room.getArray("streams").size(); s++) {
+            final JsonObject stream = room.getArray("streams").getObject(s);
+            final JsonObject urls = stream.getObject("urls");
+            if (urls.has("dash")) {
+                return urls.getObject("dash").getString("url");
+            }
+        }
+        return EMPTY_STRING;
     }
 
+    /**
+     * Get the URL of the first HLS stream found.
+     * <p>
+     * There can be several HLS streams, so the URL of the first found is returned by this method.
+     * <br>
+     * You can find the other video HLS streams
```

Comment:
where?

Reply:
Sorry, I forgot to finish the JavaDoc. Fixed.